src/HornedBeast.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+class HornedBeast extends React.Component {
+  render () {
+    return (
+      <>
+      </>
+    )
+  }
+}
```

Comment:
- We are missing some pieces here (see lab instructions for HornedBeast component 👍 
- This component needs to contain:
     - an `<h2>` that displays the `title` of the animal, an `<img>` element with `src`, `alt` and `title` attributes
     -  a `<p>` that displays the description.